Objects/Converters/ConverterRevit/ConverterRevitShared/ConverterRevit.cs

```diff
@@ -408,9 +408,26 @@ public object ConvertToNative(Base @object)
         // find self referential prop and set value to @object if it is null (happens when sent from gh)
         if (CanConvertToNative(speckleSchema))
         {
-          var prop = speckleSchema.GetInstanceMembers().Where(o => speckleSchema[o.Name] == null)?.Where(o => o.PropertyType.IsAssignableFrom(@object.GetType()))?.FirstOrDefault();
-          if (prop != null)
-            speckleSchema[prop.Name] = @object;
+          //if you can find a "MainParamProperty" get that
```

Comment:
minor, but should remove this comment

Reply:
oopsies! thanks!

Reply:
did one better and moved the whole thing to it's own function, plus moved around comments so that they'd make sense